examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Inc/mems_cube_hal.h

```diff
@@ -0,0 +1,121 @@
+/**

```

Comment:
are demo_io.h and mems_cube_hal.h files from the mems SDK? Are they intact? If we use the "stock" version than we shouldn't put it into our sources but link it from the mems SDK. Probably we already have to have a variable containing the path to mems SDK, don't we?
I know that Milan compied all the files into current wifi examples. I just removed it in recent stories and referencing to files in the Wifi SDK.